samples/extensions/mesh_shader_culling/CMakeLists.txt

```diff
@@ -0,0 +1,31 @@
+# Copyright (c) 2023, Holochip Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 the "License";
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+get_filename_component(FOLDER_NAME ${CMAKE_CURRENT_LIST_DIR} NAME)
+get_filename_component(PARENT_DIR ${CMAKE_CURRENT_LIST_DIR} PATH)
+get_filename_component(CATEGORY_NAME ${PARENT_DIR} NAME)
+
+add_sample_with_tags(
+        ID ${FOLDER_NAME}
+        CATEGORY ${CATEGORY_NAME}
+        AUTHOR "Holochip Corporation"
+        NAME "Mesh Shader Culling"
+        DESCRIPTION "Demonstrate and showcase a sample application using mesh shader culling functionality"
+        SHADER_FILES_GLSL
+        "mesh_shader_culling/mesh_shader_culling.task"
+        "mesh_shader_culling/mesh_shader_culling.mesh"
+        "mesh_shader_culling/mesh_shader_culling.frag")
```

Comment:
I went ahead and did this; however, we should have a talk in the group about best practices for header files used only in shaders being included in CMake.
CMake's normal rules are that you don't include header files in your targets, they are discovered by the build tools and CMake does the right thing from there.  However, CMake can't do the right thing in shader includes because no tooling exists that will discover and inform CMake about that header file.  We might even take it so far as to have a talk about if we want to introduce a best practice of providing the shader includes with a different extension so tooling can potentially do syntax highlighting and it will be easier to write other tools like static analysis.